westcoast/src/main/java/frc/commands/ActuateTwoDoubleSolenoids.java

```diff
@@ -0,0 +1,81 @@
+package frc.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+public class ActuateTwoDoubleSolenoids extends Command {
+  private final DoubleSolenoid solenoid;
+  private final DoubleSolenoid solenoid2;
+	private final Value direction;
+  private final Value endDirection;
+  public enum state {
+    bothout, flapperout, pusherout, bothin
+  }
```

Comment:
`Enums` are considered `Types` , and typical naming convention for `Types` in Java follows `PascalCase` style.
Now elements have more style variety, but overall each word in the is somewhat identifiable and they are considered `constants`/`literals` so typically convention is either `PascalCase` or `SCREAMING_SNAKE_CASE` (which is the Java standard).
```suggestion
  private enum State {
    BOTH_OUT, FLAPPER_OUT, PUSHER_OUT, BOTH_IN
  }
```